lmdk/include/adsp_stddef.h

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
We need to set up our cmake to include from the global include directories to avoid the duplication